core/lib/zksync_core/src/metadata_calculator/updater.rs

```diff
@@ -204,14 +206,14 @@ impl TreeUpdater {
             .await
             .unwrap();
 
-        let is_pre_boojum = header
+        // TODO(X): ensure that the protocol version is always available.
```

Comment:
is this TODO still needed?

Reply:
cc @perekopskiy ?

It is not needed for this PR, but I thought that maybe we do have plans to do it in the future

Reply:
We definitely have these plans, I'm just not sure when we will capacity to implement them